src/snowflake/cli/plugins/snowpark/common.py

```diff
@@ -50,6 +52,9 @@ def check_if_replace_is_required(
         )
         return True
 
+    if [imp.lower() for imp in imports] != _get_import_from_resource(resource_json):
```

Comment:
Is the order always the same? Should we perform set operation?

Reply:
Good point. Changed